### ISSUE TITLE
where are set paths and urls?

### ISSUE THREAD

**Issue by JohnKey9** [StartupAPI/users#290]
hello,

I wonder, where in the code can I find
Paths and URLs that are set automatically?
ie
Root URL of Startup API code (full, e.g. http://example.com/myapp/users/)
code  UserConfig::$USERSROOTFULLURL (string)

or
/**
 * Default location URL to return to upon login
 *
 * @var string
 */
UserConfig::$DEFAULTLOGINRETURN = '/';

in which file is this code please?

many thanks

**Comment by JohnKey9** [comment]
I have found a solution for this.

the two main relevant files are default_config.php and users_config.php
pls, note they are in different folders

**Comment by sergeychernyshev** [comment]
As you probably figured out already, `default_config.php` is automatically set configurations that can be changed in `users_config.php`

You should only change `users_config.php` file.

It's good to look at default one if you need to know all the settings that can be set in `users_config.php`

**Comment by JohnKey9** [comment]
tnx a lot,

that is why I asked... I thought I was doing something wrong... in fact I had to change also default...

pls, let me verify asking, if.. I have to change

/**
 * Default location URL to return to upon login
 *
 * @var string
 */
UserConfig::$DEFAULTLOGINRETURN = 'users/user_loggedin/loggedinpage.html';

as per tab `settings` for the `admin` user.
How can I set this in the users_config.php?
should I just copy the code and paste at the end of the file?
I had tried but did not work, maybe there is something else I need to set up.

many thanks
